opendrift/models/radionuclides.py

```diff
@@ -0,0 +1,620 @@
+                    6:{'name':'Sediment slowly reversible','color':'lightred'},
+                    7:{'name':'Sediment irreversible','color':'darkred'}
+                    }
+    
+    configspec_radionuclidedrift = '''
+        [radionuclide]
+            transfer_setup = option('Bokna_137Cs', 'dummy', default='dummy')
+            slowly_fraction = boolean(default=False)
+            irreversible_fraction = boolean(default=False)
+            dissolved_diameter = float(min=0., max=100.e-6,default=0.)
+            particle_diameter = float(min=0., max=100.e-6,default=5.e-6)
+            particle_diameter_uncertainty = float(min=0., max=100.e-6, default=1.e-7)
+            [[species]]
+                LMM                        = boolean(default=False)
+                Colloid                    = boolean(default=False)
+                Particle_reversible        = boolean(default=False)
+                Particle_slowly_reversible = boolean(default=False)
+                Particle_irreversible      = boolean(default=False)
+                Sediment_reversible        = boolean(default=False)
+                Sediment_slowly_reversible = boolean(default=False)
+                Sediment_irreversible      = boolean(default=False)
+            [[transformations]]
+                Kd         = float(min=0., max=1.e9, default=0.)
+                Dc         = float(min=0., max=1.e6, default=1.16e-5)
+                slow_coeff = float(min=0., max=1.e6, default=1.2e-7)
+            [[sediment]]
+                sedmixdepth        = float(min=0., max=100., default=1.)
+                sediment_density   = float(min=0., max=10000., default=2600.)
+                effective_fraction = float(min=0., max=1., default=0.9)
+                corr_factor        = float(min=0., max=10., default=0.1)
+                porosity           = float(min=0., max=1., default=0.6)
+                layer_thick        = float(min=0., max=100., default=1.)
+                desorption_depth   = float(min=0., max=100., default=10.)
+                desorption_depth_uncert = float(min=0., max=100., default=1.)
+            '''
+
+
+    def specie_num2name(self,num):
+#        return self.species_all[num]['name']
+        return self.name_species[num]
+
+
+
+    def specie_name2num(self,name):
+        #kktmp = np.where([self.species_all[i]['name']==name for i in self.species_all])[0]
+        num = self.name_species.index(name)
+        return num
+#       
+
+
+
+    def __init__(self, *args, **kwargs):
+
+        self._add_configstring(self.configspec_radionuclidedrift)
+
+        # Calling general constructor of parent class
+        super(RadionuclideDrift, self).__init__(*args, **kwargs)
+
+        # By default, eggs do not strand towards coastline
+        self.set_config('general:coastline_action', 'previous')
+        
+
+        
+    
+    
+        
+
+
+    def prepare_run(self):
+        print 'Preparing'
```

Comment:
Won't work on python3

Reply:
I suggest to replace the print-line with:
```logging.debug('Preparing radionucleide species')```